ddtrace/bootstrap/sitecustomize.py

```diff
@@ -119,6 +120,9 @@ def update_patched_modules():
         env_tags = os.getenv("DD_TRACE_GLOBAL_TAGS")
         tracer.set_tags(parse_tags_str(env_tags))
 
+    if asbool(get_env("instrumentation_telemetry", "enabled")):
```

Comment:
is there a specific reason why it is here and not further up? (e.g. if the ordering matters, this must have a comment)

Reply:
I added a comment. We just need to start the telemetry service after `from ddtrace import tracer`. Once the global tracer and configurations are initialized we're good to go

Reply:
just ddtrace.config